src/plugins/content_management/public/rpc_client/rpc_client.ts

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { HttpSetup } from '@kbn/core/public';
+import { API_ENDPOINT } from '../../common';
+import type { GetIn, CreateIn, ProcedureName } from '../../common';
+
+export class RpcClient {
```

Comment:
This file is copy paste from the server-side POC https://github.com/elastic/kibana/pull/148791/. 

Assuming this won't change much when initial server-side is merged to main. Also won't be a problem if this is changed a lot